<ui version="4.0" >
 <class>statusDialogClass</class>
 <widget class="QDialog" name="statusDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>statusDialog</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="3" >
    <widget class="QCheckBox" name="dontShowBox" >
     <property name="text" >
      <string>Don't show this dialog</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QTextEdit" name="statusEdit" />
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="closeAfter" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2" >
    <widget class="QPushButton" name="okButton" >
     <property name="text" >
      <string>OK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>statusDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>282</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel" >
     <x>194</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
